timesketch/frontend-ng/src/views/Studio.vue

```diff
@@ -0,0 +1,147 @@
+<!--
+Copyright 2019 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+    <div>
+        <!-- Left panel -->
```

Comment:
I know this is still draft but doing a drive-by early comment here.

This is copied from Sketch.vue. We do not want to duplicate the menu here. Do we need the menu at all in the studio? If so, can we put this under the sketch, and reuse the side panel from there?

IMO, the Studio can be its own thing, using the full width of the UI. Let's discuss in person.

Reply:
we talked about it and as far as I can tell we settled to use the left menu, closing that comment